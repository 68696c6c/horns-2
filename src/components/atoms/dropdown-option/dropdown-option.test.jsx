/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import DropdownOption from '.'

describe('DropdownOption', () => {
  it('should render as default', () => {
    const { container } = render(
      <ul>
        <DropdownOption>a</DropdownOption>
        <DropdownOption>b</DropdownOption>
      </ul>
    )
    expect(container).toMatchSnapshot()
  })
})
