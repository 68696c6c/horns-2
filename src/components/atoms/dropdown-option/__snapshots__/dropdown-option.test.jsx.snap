// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownOption should render as default 1`] = `
.emotion-0 {
  background: rgb(255,255,255);
  color: rgb(1,1,1);
  border-color: rgb(216,216,216);
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 8px;
  padding-right: 8px;
}

<div>
  <ul>
    <li
      class="emotion-0"
      color="background"
      cursor="pointer"
    >
      a
    </li>
    <li
      class="emotion-0"
      color="background"
      cursor="pointer"
    >
      b
    </li>
  </ul>
</div>
`;
